storages/backends/base.py

```diff
@@ -28,15 +28,15 @@ def size(self, name: str) -> int:
         pass  # pragma: no cover
 
     @abstractmethod
-    def get_created_time(self, name: str) -> datetime:
+    def get_created_time(self, name: str) -> Optional[datetime]:

```

Comment:
Why `Optional`? Google Cloud Storage provides the creation time for each object. What is the scenario for returning `None` here?

Reply:
On Google Cloud to get `created_time` value we have to get the blob first for a given name. If the user pass not existing name of the file then we will get a None here and this won't be a correct datetime so it's why I added Optional here. 

If you have better idea let me know :)

Reply:
I can see that `django-storages` is raising an exception if blob is none
```
def _get_blob(self, name):
        # Wrap google.cloud.storage's blob to raise if the file doesn't exist
        blob = self.bucket.get_blob(name)

        if blob is None:
            raise NotFound('File does not exist: {}'.format(name))

        return blob
```
https://github.com/jschneier/django-storages/blob/cf50a22d76f1ab3294e85c0a753a7c369663b293/storages/backends/gcloud.py#L250